typescript/packages/orb/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@subconsciousnetwork/orb",
+  "author": "Subconscious Inc.",
+  "license": "Apache-2.0 OR MIT",
+  "version": "0.1.0",
+  "description": "A batteries-included JavaScript package for accessing the Noosphere",
+  "type": "module",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "wireit",
+    "generate_wasm": "wireit",
+    "clean": "wireit",
+    "test": "wireit"
+  },
+  "dependencies": {
+    "@web/dev-server": "^0.1.34",
+    "esbuild": "^0.15.9",
+    "typescript": "~4.8.2",
+    "wireit": "^0.8.0"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "generate_wasm"
+      ],
+      "command": "tsc",
+      "files": [
+        "./src/**/*.ts"
+      ]
+    },
+    "generate_wasm": {
+      "command": "./scripts/generate-wasm-artifacts.sh",
+      "files": [
+        "../../../rust/Cargo.toml",
+        "../../../rust/**/Cargo.toml",
+        "../../../rust/**/*.rs",
+        "./scripts/genergate-wasm-artifacts.sh"
+      ],
+      "output": [
+        "lib/noosphere*.{ts,wasm,js}"
+      ]
+    },
+    "clean": {
+      "command": "rm -rf ./lib && rm -f ./tsconfig.tsbuildinfo"
+    },
+    "test": {
+      "dependencies": [
+        "build"
+      ],
+      "command": "web-test-runner --node-resolve"
```

```suggestion
      "command": "web-test-runner --node-resolve",
      "files": ["web-test-runner.config.js"],
      "output": []
```

This way if no input files changed, a previously passing result will be remembered, and the test won't need to run again until you change something.

The `"output": []` is required because wireit treats "unknown output files" differently from "definitely no output files".